src/_P097_Esp32Touch.ino

```diff
@@ -77,16 +78,25 @@ boolean Plugin_097(uint8_t function, struct EventStruct *event, String& string)
     }
 
     case PLUGIN_GET_DEVICEVALUENAMES:
-    {
+    { 
       strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_097));
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], PSTR(PLUGIN_VALUENAME2_097));
+      ExtraTaskSettings.TaskDeviceValueDecimals[1] = 0;
+      ExtraTaskSettings.TaskDeviceValueDecimals[2] = 0;
```

Comment:
I assume this should be indexes 0 and 1, not 1 and 2 🤔

Reply:
You are right...